modules/evm-bridge/src/tests.rs

```diff
@@ -380,33 +381,35 @@ fn tracing_should_work() {
 					Error::<Runtime>::ExecutionRevert
 				);
 			});
-			let expected = r#"[
-			  {
-				"type": "CALL",
-				"from": "0x1000000000000000000000000000000000000002",
-				"to": "0x5dddfce53ee040d9eb21afbc0ae1bb4dbb0ba643",
-				"input": "0xa9059cbb0000000000000000000000001000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000a",
-				"value": "0x0",
-				"gas": 200000,
-				"gasUsed": 200000,
-				"output": null,
-				"error": null,
-				"revertReason": "0x08c379a00000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000002645524332303a207472616e7366657220616d6f756e7420657863656564732062616c616e63650000000000000000000000000000000000000000000000000000",
-				"depth": 0,
-				"logs": [
-				  {
-				    "sLoad": {
-				      "address": "0x5dddfce53ee040d9eb21afbc0ae1bb4dbb0ba643",
-				      "index": "0xfb750de6f7d0583f749efc558ce6626b24fed04efd7219dc3f4294c408699e8c",
-				      "value": "0x0000000000000000000000000000000000000000000000000000000000000000"
-				    }
-				  }
-				],
-				"calls": []
-			  }
-			]"#;
-			let expected = serde_json::from_str::<Vec<tracing::CallTrace>>(expected).unwrap();
-			assert_eq!(tracer.finalize(), tracing::TraceOutcome::Calls(expected));
+			assert_json_snapshot!(tracer.finalize(), @r###"
+   {
+     "Calls": [
+       {
+         "type": "CALL",
+         "from": "0x1000000000000000000000000000000000000002",
+         "to": "0x5dddfce53ee040d9eb21afbc0ae1bb4dbb0ba643",
+         "input": "0xa9059cbb0000000000000000000000001000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000a",
+         "value": "0x0",
+         "gas": 200000,
+         "gasUsed": 200000,
+         "output": null,
+         "error": null,
+         "revertReason": "0xe450d38c00000000000000000000000010000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a",
```

Comment:
@zjb0807 revert reason changed. can you confirm if this is ok?

Reply:
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/dc44c9f1a4c3b10af99492eed84f83ed244203f6/contracts/token/ERC20/ERC20.sol#L229

https://github.com/OpenZeppelin/openzeppelin-contracts/blob/dbb6104ce834628e473d2173bbc9d47f81a9eec3/contracts/token/ERC20/ERC20.sol#L195

`openzeppelin-contracts` dependency version has been updated and the restore message has changed